Calendar/BookingHelpers/ExcelWrite.ps1

```diff
@@ -0,0 +1,123 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# ========================
+# Excel needs Scriptlets helper functions to be loaded, in order to have access to graph and install Export-Excel module.
+# ========================
+
+. $PSScriptRoot\GraphInstaller.ps1
+. $PSScriptRoot\BookingGenericFunctions.ps1
+. $PSScriptRoot\BookingMBHelpers.ps1
+. $PSScriptRoot\BookingStaffHelpers.ps1
+. $PSScriptRoot\BookingServiceHelpers.ps1
+. $PSScriptRoot\BookingCustomQuestionHelpers.ps1
+. $PSScriptRoot\BookingStaffLogHelper.ps1
+
+function GetExcelParams($Path, $TabName) {
+
+    return @{
+        Path                    = $Path
+        FreezeTopRow            = $true
+        Verbose                 = $false
+        TableStyle              = "Medium3"
+        WorksheetName           = $TabName
+        TableName               = $TabName
+        FreezeTopRowFirstColumn = $false
+        AutoFilter              = $true
+        AutoNameRange           = $true
+        Append                  = $true
+        AutoSize                = $true
+        Title                   = "Bookings Support Data Collector"
+        ConditionalText         = $ConditionalFormatting
+    }
+}
+
+function ExcelWrite {
+    param($Identity)
+
+    Write-Host -ForegroundColor Gray "Checking Excel, Graph and bookings modules"
```

Comment:
This doesn't line up with the function name. Might need to change that up to make it make sense.